### PR TITLE
Update sdxc_formatting.md to include MBR scheme for Mac

### PR DESCRIPTION
**Pull request by binarygeek**
added note on which scheme to use. No where did it state that MBR was needed. I tried "GUID Partition Map (Table)" and NOOBS was unable to boot with errors

### CODE REVIEW

**Comment by JamesH65** [comment]
I have limited knowledge of the Mac, so any comments from others? Seems OK to me.

**Comment by lurch** [comment]
pin @aallan

File: installation/sdxc_formatting.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Run the Raspberry Pi Imager application, then from the `Choose OS option`, selec
 
 The standard formatting tools built into these operating systems are able to create FAT32 partitions; they might also be labelled as FAT or MS-DOS. Simply delete the existing exFAT partition and create and format a new FAT32 primary partition, before proceeding with the rest of the [NOOBS instructions](noobs.md). On a Mac, this means using the command-line `diskutil` program.
 
+NOTE: For Mac, use `Master Boot Record` for the scheme.
```
Comment (andrum99) [inline installation/sdxc_formatting.md:19]: Probably best to make this part of the preceding paragraph. Something like "On a Mac, this means using the command-line `diskutil` program: choose the `Master Boot Record` scheme."

Comment (JamesH65) [reply installation/sdxc_formatting.md:19]: I agree, think it would scan better if it were in the preceding paragraph.

**Pull request merged**